tf2onnx/onnx_opset/tensor.py

```diff
@@ -637,6 +637,161 @@ def version_4(cls, ctx, node, **kwargs):
             ctx.copy_shape(node.output[0], cast_node.output[0])
             nodes.append(cast_node)
 
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+        # T output = Slice(T input, Index begin, Index end, Index strides
+        #                 @int begin_mask, @int end_mask, @int ellipsis_mask
+        #                 @int shrink_axis_mask, @int new_axis_mask)
+        # T output = Slice(T input, Tind starts, Tind ends, Tind axes, Tind steps)
+        # "ends" are exclusive, "axes" and "steps" are optional, their default val are [0, ...] and 1
+        begin = node.inputs[1]
+        end = node.inputs[2]
+        strides = node.inputs[3]
+        strides_val = strides.get_tensor_value()
```

Comment:
strides is not always const?    the code below has statement "if strides.is_const():"